android/TritonClient/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.mobiledgex.tritonclient">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
```

Comment:
For future :

In unity, We did everything we can to not enforce READ_PHONE_STATE in the SDK.
I understand that it's different since you are building an application, not a library.
I assume this client will behave the same as the current MobiledgeXSDKDemo in which the application won't start if I didn't "Allow MobiledgeX SDK Demo to make and manage phone calls".

1. Do you need READ_PHONE_STATE or Just the telephony manager?
2. Is it worth it in the future to create a mode READ_PHONE_STATE_NOT_ENABLED, similar to EnahncedLocation not Enabled?

Reply:
I would think this is more of an SDK decision, and refer to @lgarner for his opinion.

Reply:
Yes, there is a permissions guard before usage in the library. But to not include it means you can't ask for it later. 

In a sense, the permission checker is merely a utility code call away, though the example we have asks for "all fo them". If you don't ask for it, and don't use the permission...it's fine. Just need to put in a "subscribe" permission set interface into the example.